descriptors/ModuleDescriptor-template.json

```diff
@@ -263,6 +263,37 @@
       ]
     },
     {
+      "id": "acquisitions-units-storage.memberships",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/acquisitions-units-storage/memberships",
+          "permissionsRequired": ["acquisitions-units-storage.memberships.collection.get"]
```

Comment:
Please take a look at the comment https://github.com/folio-org/mod-orders-storage/pull/132/files/407d69f2939a87adcf8c256ee7244cc900ceb1b6#r297103612
According to it permission naming should be changed to `orders-storage.acquisitions-unit-memberships.XYZ`

Reply:
Potentially this API might be moved to its own repository one day. So personally I am okay with current naming pattern.